src/features/shared-link-settings-modal/AllowDownloadSection.js

```diff
@@ -35,23 +38,38 @@ const AllowDownloadSection = ({
         </div>
     );
 
+    const label = classification ? `"${classification}"` : null;
+    const tooltipMessage = { ...messages.directDownloadBlockedByAccessPolicy, values: { label } };
+    const downloadWrapperClass = isDirectLinkUnavailableDueToAccessPolicy
+        ? 'bdl-AllowDownloadSection-wrapper--disabled'
```

Comment:
We normally try to avoid words like "wrapper" or "container" unless absolutely needed as they don't add any additional value to the CSS class.

Also in this case, you want to use `bdl-is-disabled`, as this is a disabled state vs -- which is for modifiers. One way to distinguish is when class is being added based on a condition, then its likely state. If you are permanently changing some CSS rule, then its a modifier.

Reply:
Nit: I would also recommend using `classnames` library.

Reply:
Thanks!